spec/plugin.spec.js

```diff
@@ -221,4 +221,48 @@ describe('ManifestPlugin', function() {
       });
     });
   });
+
+  describe('assets added by third parties to "compilation.assets"', function() {
+    it('are included also when enabled.', function(done) {
+
+      var options = webpackConfig({
```

Comment:
Did this since could not find a way to use mock plugin only for one test. Now it is added to all, directly in initial configuration.